index.html

```diff
@@ -3,290 +3,290 @@
 
 <head>
   <meta charset="utf-8">
-  <title>subject>Shop of ice-cream "Глейси"</title>
+  <title>Shop of ice-cream "Глейси"</title>
+  <link rel="stylesheet" href="styles/styles.css">
+  </link>
```

Comment:
Здесь я неправильно подсказал, прошу прощения, забылось. В тот раз мы забыли просто `>` добавить, поэтому была ошибка валидационная

То есть должно быть

```suggestion
  <link rel="stylesheet" href="styles/styles.css">
```